packages/askui-nodejs/src/execution/inference-client.ts

```diff
@@ -7,18 +7,24 @@ import { resizeBase64ImageWithSameRatio } from '../utils/transformations';
 import { IsImageRequired } from './is-image-required-interface';
 import { InferenceResponseError } from './inference-response-error';
 import { DetectedElement } from '../core/model/annotation-result/detected-element';
+import { ConfigurationError } from './config-error';
 
 export class InferenceClient {
   url: string;
 
   constructor(
     public baseUrl: string,
     public httpClient: HttpClientGot,
+    public resize?: number,
     readonly workspaceId?: string,
     public apiVersion = 'v3',
   ) {
     const versionedBaseUrl = urljoin(this.baseUrl, 'api', this.apiVersion);
     this.url = workspaceId ? urljoin(versionedBaseUrl, 'workspaces', workspaceId) : versionedBaseUrl;
+    if (this.resize !== undefined && this.resize <= 0) {
+      throw new ConfigurationError(`Resize must be a postive number. the current resize value "${this.resize}" is not valid.`);
```

Comment:
Early return :sunglasses: 

```suggestion
      throw new ConfigurationError(`Resize must be a positive number. The current resize value "${this.resize}" is not valid.`);
```